test/e2e/helpers.js

```diff
@@ -143,10 +143,10 @@ async function withFixtures(options, testSuite) {
 
     await testSuite({
       driver: driverProxy ?? driver,
-      mockServer,
       contractRegistry,
       ganacheServer,
       secondaryGanacheServer,
+      mockedEndpoint,
```

Comment:
instead of sharing the `mockServer` with the test, we share `mockedEndpoint` which is the result of `testSpecificMock`. This allows us to continue making assertions on specific requests in the test